app/assets/stylesheets/_loading_indicator.sass

```diff
@@ -0,0 +1,77 @@
+$half-inner-balance-height: 0.312rem
+$loader-height: $feed-item-height - (2 * $feed-item-padding-vertical) - $half-inner-balance-height
+
+.feed-wrapper
+  .feed
+    .item
+.cs-loader
```

Comment:
`.loading-indicator` of een andere naam, zolang het react component (nu LoadingIndicator) en de css-klasse maar overeen komen